Basim_Muhammad/animal.py

```diff
@@ -0,0 +1,47 @@
+class Animal(object):
+	def __init__(self,name):
+		self.name=name
+		self.health=100
+	def walk(self):
+		self.health-=1
+		return self
+	def run(self):
+		self.health-=5
+		return self
+	def displayHealth(self):
+		print self.name, self.health
+		return self
+
+class Dog(Animal):
+	def __init__(self,name):
+		super(Dog,self).__init__(name)
+		self.health=150
+	def pet(self):
+		self.health+=5
+		return self
+
+class Dragon(Animal):
+	def __init__(self,name):
+		super(Dragon,self).__init__(name)
+		self.health=170
+	def fly(self):
+		self.health-=10
+		return self
+	def displayHealth(self):
+		print 'this is a dragon',self.name, self.health
```

Comment:
The Animal class already has a displayHealth method that prints self.name and self.health.  You should be using 'super' to call on that original method after adding the "print 'this is a dragon'"